facia/app/services/dotcomrendering/FaciaPicker.scala

```diff
@@ -172,9 +96,9 @@ object FaciaPicker extends GuLogging {
     if (isRss) LocalRender
     else if (forceDCROff) LocalRender
     else if (forceDCR) RemoteRender
-    else if (isNetworkFront)
+    else if (isNetworkFront) {
```

Comment:
```suggestion
    else if (isNetworkFront && dcrSwitchEnabled) {
```

As is I think turning off the `dcr-fronts` switch wouldn't disable the `dcr-network-fronts` experiment from rendering fronts in DCR. Is this behaviour we want to keep?

Reply:
Ah good point, but I think we would have to have that check inside this if, because otherwise you could still have `isNetworkFront = true` in the `else` in case the switch is turned off.